tests/test_partnerships.py

```diff
@@ -10,26 +10,6 @@
 class TestPartnerships:
 
     @pytest.mark.nondestructive
-    def test_section_link_destinations_are_correct(self, base_url, selenium):
-        page = Partnerships(base_url, selenium).open()
-        bad_links = []
-        for link in page.section_links_list:
-            url = page.link_destination(link.get('locator'))
-            if not url.endswith(link.get('url_suffix')):
-                bad_links.append('%s does not end with %s' % (url, link.get('url_suffix')))
-        assert [] == bad_links
-
-    @pytest.mark.nondestructive
-    def test_image_srcs_are_correct(self, base_url, selenium):
-        page = Partnerships(base_url, selenium).open()
-        bad_images = []
-        for image in page.images_list:
-            src = page.image_source(image.get('locator'))
-            if not image.get('img_name_contains') in src:
-                bad_images.append('%s does not contain %s' % (src, image.get('img_name_contains')))
-        assert [] == bad_images
-
-    @pytest.mark.nondestructive
     def test_partner_form_is_visible(self, base_url, selenium):
```

Comment:
Sure, I'm planning on migrating it (unless you get to it first) to get even closer to removing mcom-tests. :)